modin/experimental/engines/omnisci_on_ray/frame/partition_manager.py

```diff
@@ -30,26 +32,63 @@
 
 
 class OmnisciOnRayFrameManager(RayFrameManager):
-    """This method implements the interface in `BaseFrameManager`."""
+    """
+    Frame manager for `OmnisciOnRayFrame`.
+
+    This class handles several features of `OmnisciOnRayFrame`:
+      - frame always has a single partition
+      - frame cannot process some data types
+      - frame has to use mangling for index labels
+      - frame uses OmniSci engine for execution
+    """
 
-    # This object uses RayRemotePartition objects as the underlying store.
     _partition_class = OmnisciOnRayFramePartition
     _column_partitions_class = OmnisciOnRayFrameColumnPartition
     _row_partition_class = OmnisciOnRayFrameRowPartition
 
     @classmethod
     def _compute_num_partitions(cls):
-        """Currently, we don't handle partitioned frames for OmniSci engine.
-        Since we support a single node mode only, allow OmniSci perform
-        partitioning by itself.
+        """
+        `OmnisciOnRayFrame` always has a single partition.
```

Comment:
This looks more like a `Notes`, not a summary.

Reply:
Added another summary.

Reply:
```suggestion
        ``OmnisciOnRayFrame`` always has a single partition.
```